README.md

```diff
@@ -44,6 +44,10 @@ To learn more about our monthly feature releases and yearly major releases, see
 
 We'd love to have your help developing Chef Infra. See our [Contributing Document](./CONTRIBUTING.md) for more information on getting started.
 
+## How to use [Dobi](https://dnephin.github.io/dobi/)
```

Comment:
This should probably just be a Building Docker Container doc here: https://github.com/chef/chef/tree/master/docs/dev